docs/Tutorial_DAMP.ipynb

```diff
@@ -0,0 +1,885 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #37.        IL[:split_idx] = -1  # <code>split_idx</code> or <code>split_idx-m</code>?
</pre><p> </p><p>I added this comment for our future reference. In the paper, a subsequence is compared with all subsequences in <code>T[:split_idx]</code> , hence, the latest subsequence in the aforementioned chunk is a subsequence that starts at location <code>split_idx -  m</code>. Although <code>excl_zone</code> is set to <code>m</code> in the paper,  the value <code>split_idx - m</code> should still remain the same even with different excl_zone  [IMO].</p><p> </p><p>Should we think about how data arrives? [e.g. In batch of size of <code>m</code> , etc]   </p><p>@NimaSarajpoor: Read the paper regarding this matter. [note to self]</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/872/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
<p>To avoid this in <code>naive_DAMP</code>  and to improve readability, the manipulation of IL is avoided. Instead, we just start computing PL from <code>split_idx</code>  <span class="ql-cursor">﻿</span></p>